cumulus/parachains/runtimes/people/people-rococo/src/lib.rs

```diff
@@ -430,7 +456,7 @@ construct_runtime!(
 		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 30,
 		PolkadotXcm: pallet_xcm::{Pallet, Call, Event<T>, Origin, Config<T>} = 31,
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Event<T>, Origin} = 32,
-		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 33,
```

Comment:
The `DmpQueue` should still stay in for one release, since it does a lazy migration if there are any pending XCMPs.

Reply:
But the parachain will be at genesis. How would there be any pending messages to migrate?

Reply:
Oh, 🤦 i just saw the diff and though it existed already.